src/main/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -291,8 +293,7 @@ public boolean isQueued(int taskId)
 	@Override
 	public @NotNull List<BukkitTask> getPendingTasks()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return tasks.stream().filter(task -> task.getTaskId() != -1).collect(Collectors.toList());
```

Comment:
This implementation is not correct, `getPendingTasks()` should be thread-safe and include asynchronous tasks as well.

Reply:
I've tried to implement that in a new version, now also not finished async task should be returned.